### DISCUSSION TITLE
Backing up Azure SQL

### DISCUSSION THREAD

**Discussion by wyeack** [olahallengren/sql-server-maintenance-solution#295] [Questions & Answers]
Your docs mention that you support backing up Azure SQL Managed Instance (https://ola.hallengren.com/sql-server-backup.html) - how does that work?  I don't see any parameters in the docs that would seem to take the URL for the Azure SQL Instance.

**Comment by liepinshw** [comment]
@wyeack are you looking for this? :
in here - https://ola.hallengren.com/

Striping of backups to Azure Blob Storage
The SQLCAT - team has a blog post about how you can optimize performance when backing up to Azure Blob Storage. You can stripe the backups to multiple files, and use the options MAXTRANSFERSIZE, and BLOCKSIZE.
EXECUTE dbo.DatabaseBackup @Databases = 'USER_DATABASES',
@URL = 'https://myaccount.blob.core.windows.net/mycontainer',
@BackupType = 'FULL',
@Compress = 'Y',
@Verify = 'Y',
@NumberOfFiles = 8,
@MaxTransferSize = 4194304,
@BlockSize = 65536
